built-in-references/Kubernetes/allowed-external-ips/template.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: k8sazureexternalips
+spec:
+  crd:
+    spec:
+      names:
+        kind: K8sAzureExternalIPs
+      validation:
+        # Schema for the `parameters` field
+        openAPIV3Schema:
+          properties:
+            allowedIPs:
```

Comment:
Rename 'allowedIPs' to 'allowedExternalIPs'. The parameter name here should match with that of the parameter name specified in constraint.yaml.

Reply:
done